packages/core/src/components/dropdown/dropdown.tsx

```diff
@@ -81,12 +88,22 @@ export class Dropdown {
    */
   @Prop() header?: string;
 
+  /**
+   * Move dropdown along main axis of alignment
```

Comment:
Rather unclear description :)

Reply:
https://floating-ui.com/docs/offset#options

Any other suggestions?

Reply:
I understand this is the floating UI API but it seems a little strange to have this object hash with only one property.

Reply:
Its only an internal prop its not exposed in documentation in some way. I can copy the complete obejct definition?

I dont want to reuse the typing of the floating lib because then we have a technical dependency in our components definition which is also not the best

Reply:
If this is the only property we are using and it is not part of the API I think we can leave it like this.